company.el

```diff
@@ -3218,10 +3218,10 @@ If SHOW-VERSION is non-nil, show the version in the echo area."
                   'company-scrollbar-fg
                 'company-scrollbar-bg)))
 
-(defun company--scrollpos-line (text width)
+(defun company--scrollpos-line (text width right-margin-width)
```

Comment:
Perhaps `left` rather than `right`?

Looks good otherwise.

Reply:
Yes, I thought so initially and the initial implementations used the following approaches, none of which worked fine in my opinion. The current solution has its drawback too (confusing change of names), but reflects correctly the use of the width-value: we pass left-margin-width to be used as a right-margin-width. I'd be happy to improve relatively to the described issues, but nothing better comes to my mind at the moment.
1. UI didn't look nice.
```
(concat (company-space-string left-margin-width)
                       (company-safe-substring text 0 width)
                       (company-space-string company-tooltip-margin)))
```
2. Lost flexibility of configuring the right margin specifically (probably in the future).
```
(concat (company-space-string company-tooltip-margin)
                       (company-safe-substring text 0 (+ width left-margin-width)))
```
3. Confusingly looking internals of `company--scrollpos-line` function: why do we add left-margin to the right side of the line?
```
(concat (company-space-string company-tooltip-margin)
                       (company-safe-substring text 0 width)
                       (company-space-string left-margin-width)))
```

Reply:
Fair enough, thank you.

We can rename it to `extra-margin-width` sometime later, I guess.

Reply:
Sure. And thank you for maintaining the packages!